allure-generator/src/main/javascript/components/modal/styles.scss

```diff
@@ -18,6 +18,9 @@
     top: 0;
     z-index: 1001;
   }
+  &__close {
```

Comment:
it is better to use flex instead:

```hbs
<h2 class="{{b cls 'header'}}">
    <span class="{{b cls 'title'}}">{{title}}</span>
    <span class="{{b cls 'close'}} fa fa-close" data-tooltip="Close"></span>
</h2>
```

```scss
.modal {
  &__header {
    display: flex;
  }
  &__title {
    flex: 1;
  }
}
```

Reply:
also it would be nice to add translation for tooltip at this point. Simply replace `Close` text with the following template

```hbs
{{t 'controls.close'}}
```

and then you can add translations to `translations/*.json` files (only english is enough):

```
//en.json
...
"controls": {
   ...
   "close": "Close"
}
...
```